### PR TITLE
Adding support to tables

### PR DESCRIPTION
**Pull request by erickmartins**
## Description

Adds `post_table` and `get_table`, at least partially covering #16. An optional `pandas` dependency is added via the installation option `ezomero[tables]` - if that is available, posting a table from a pandas DataFrame and returning a table to a pandas DataFrame are enabled; otherwise, tables are posted/returned as simple lists of row-lists.

## Checklist

Docstrings, unit tests and flake8 pass done.

## For reviewers

- Check that the PR title is short, concise, and will make sense 1 year
  later.
- Check that new functions are imported in corresponding `__init__.py`.

### CODE REVIEW

File: ezomero/_gets.py [hunk 1]
```diff
@@ -1077,6 +1084,43 @@ def get_pyramid_levels(conn, image_id, across_groups=True):
...
+    >>> table = get_table(conn, 62)
+    >>> print(table[0])
+    ['ID', 'X', 'Y']
+    """
+    if type(file_ann_id) is not int:
+        raise TypeError('File annotation ID must be an integer')
+    ann = conn.getObject('FileAnnotation', file_ann_id)
+    orig_table_file = conn.getObject('OriginalFile', ann.getFile().id)
+    resources = conn.c.sf.sharedResources()
+    table_obj = resources.openTable(orig_table_file._obj)
```
Comment (mellertd) [inline ezomero/_gets.py:1119]: do we need to catch whether someone gives a file ID that is not a table?

File: ezomero/_posts.py [hunk 2]
```diff
@@ -237,7 +246,7 @@ def post_map_annotation(conn, object_type, object_id, kv_dict, ns,
     >>> post_map_annotation(conn, "Image", 56, d, ns)
     234
     """
-
+    print("group at begin", conn.getGroupFromContext().getId())
```
Comment (mellertd) [inline ezomero/_posts.py:249]: What is the significance of this line, and 276?

File: ezomero/_posts.py [hunk 3]
```diff
@@ -522,6 +531,135 @@ def post_roi(conn, image_id, shapes, name=None, description=None,
...
+
+    Returns
+    -------
+    TableFile_id : int
+        ID of newly created FileAnnotation containing the new Table.
+
+
+    Notes
+    -------
+    Currently not working with `across_groups` - the `OriginalFile` seems to
```
Comment (mellertd) [inline ezomero/_posts.py:568]: Is this worth a question to the OME team before merging?

File: ezomero/_posts.py (hunk 2 above)
Comment (erickmartins) [reply ezomero/_posts.py:249]: that was me trying to debug the across-groups issue and forgetting to remove prints :)

File: ezomero/_posts.py (hunk 3 above)
Comment (erickmartins) [reply ezomero/_posts.py:568]: I think it's worth pushing it as is and opening a new issue for that problem - we can reach out to them when working that issue

File: ezomero/_gets.py (hunk 1 above)
Comment (erickmartins) [reply ezomero/_gets.py:1119]: I think we should - good catch. I'll try and figure out how to do that.

File: ezomero/_posts.py (hunk 3 above)
Comment (erickmartins) [reply ezomero/_posts.py:566]: (in fact I had already created issue #59 lol)

**Pull request merged**